deployments/dev/environment.py

```diff
@@ -45,7 +45,7 @@ def mkdict(items):
     mksrc('datarepo-dev-a9252919', 'hca_dev_005d611a14d54fbf846e571a1f874f70__20210827_20210903', 7),
     mksrc('datarepo-dev-c148d39c', 'hca_dev_027c51c60719469fa7f5640fe57cbece__20210827_20210902', 8),
     mksrc('datarepo-dev-e2ab8487', 'hca_dev_03c6fce7789e4e78a27a664d562bb738__20210902_20210907', 1530),
-    mksrc('datarepo-dev-37639c56', 'hca_dev_05657a599f9d4bb9b77b24be13aa5cea__20210827_20210928', 185),
```

Comment:
We should add a `dcp3` catalog so that the snapshot changes are tracked the same way we do in production. The `dcp2` catalog does not need to be active.